### PR TITLE
Explain implementation of Activity Goal Tracking in DG

### PR DESCRIPTION
**Pull request by AlWo223**

### CODE REVIEW

File: docs/images/AddActivityGoal.svg
Comment (skylee03) [inline docs/images/AddActivityGoal.svg:1]: We should use `-->` instead of `->` for `message`.

File: docs/images/AddActivity.svg
Comment (skylee03) [inline docs/images/AddActivity.svg:1]: We should use `-->` instead of `->` for `message`.

File: docs/images/ActivityGoalEvaluation.svg
Comment (skylee03) [inline docs/images/ActivityGoalEvaluation.svg:1]: We should use `-->` instead of `->` for `activities`.

File: docs/images/AddActivityGoal.svg
Comment (skylee03) [inline docs/images/AddActivityGoal.svg:1]: According to the [textbook](https://nus-cs2113-ay2324s1.github.io/website/schedule/week10/topics.html#w10-2-sequence-diagrams-intermediate-level), method calls to `static` (i.e., class-level) methods are received by the class itself, not an instance of that class. You can use `<<class>>` to show that a participant is the class itself.

File: docs/images/AddActivity.svg
Comment (skylee03) [inline docs/images/AddActivity.svg:1]: According to the [textbook](https://nus-cs2113-ay2324s1.github.io/website/schedule/week10/topics.html#w10-2-sequence-diagrams-intermediate-level), method calls to `static` (i.e., class-level) methods are received by the class itself, not an instance of that class. You can use `<<class>>` to show that a participant is the class itself.

File: docs/images/AddActivityGoal.svg
Comment (AlWo223) [reply docs/images/AddActivityGoal.svg:1]: Thank you, good to know!

**Comment by skylee03** [comment]
`ActivityGoalEvaluation.puml` is updated, but the corresponding `ActivityGoalEvaluation.svg` is not.

**Review by skylee03 — APPROVED** [review]
LGTM!

**Pull request merged**